src/molecules/Footer/footer.module.css

```diff
@@ -0,0 +1,12 @@
+footer {
+  padding: 20px 28px;
```

Comment:
We need to use `calc()` and `--layout-base` here.

Reply:
Done.